Examples/Makefile.in

```diff
@@ -1252,13 +1251,16 @@ PYTHON_FLAGS =
 # Make sure these locate your Python installation
 ifneq (,$(PY2))
   PYTHON_INCLUDE= $(DEFS) @PYINCLUDE@
-  PYTHON_LIB    = @PYLIB@
+  PY_LIB        = @PYLIB@
   PYTHON        = @PYTHON@ $(PYTHON_FLAGS)
 else
   PYTHON_INCLUDE= $(DEFS) @PY3INCLUDE@
-  PYTHON_LIB    = @PY3LIB@
+  PY_LIB        = @PY3LIB@
   PYTHON        = @PYTHON3@ $(PYTHON_FLAGS)
 endif
+ifneq (,$(PY_LIB))
+  PYTHON_LIB    = -L$(PY_LIB)
+endif
```

Comment:
The problem is the '-L' in case 'PY_LIB' is empty.
The Make-file assume 'PY_LIB' must have a folder value.
configure is allow the set 'PY_LIB' to empty.
I do not think 'configure' should add '-L' to PY_LIB.

Reply:
Oh dear, this all rather horrible, but are only wrong for building static libraries and we never do that. So basically a non-issue. I think we should remove the static library option instead. The problem is the same in Ruby for RUBYLIB. Also, the `-L` **should** be set in configure.ac. Please abandon these changes from this merge request or create a new one to fixup static linking, which probably means removing it.